Prerequisite & Verification for puppet environment (#1954)
theforeman/foreman-documentation: guides/common/modules/proc_validating-puppet-environments-on-the-external-capsule-servers.adoc

> :_content-type: PROCEDURE
> 
> [id="validating-puppet-environments-on-the-external-capsule-servers_{context}"]

Reviewer: ```suggestion
[id="validating-puppet-environments-on-the-external-{smart-proxy-context}-servers_{context}"]
```

= Validating Puppet Environments on the External {SmartProxyServer}

> After upgrading the {SmartProxyServer} to {ProjectVersion}, ensure the availability of the puppet environments inside the `/etc/puppetlabs/code/environments` file of the {SmartProxyServer}.

Reviewer: ```suggestion
After upgrading the {SmartProxyServer} to {ProjectVersion}, ensure the puppet environments are available on the {SmartProxyServer}.
```

.Procedure
. To view the Puppet environments, open the `/etc/puppetlabs/code/environments` file.
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# vim /etc/puppetlabs/code/environments
----

> . If any of the puppet environments are missing, copy them back from:
> .. the {project} server or 
> .. the backup taken prior to upgrade 

Reviewer: ```suggestion
. If any of the puppet environments are missing, copy them back from the {ProjectServer} or the backup taken prior to upgrade.
```

> . Ensure to have the right permissions and ownership of the missing contents inside  `/etc/puppetlabs/code/environments/` file after they have been restored back.

Reviewer: ```suggestion
. Ensure to have the right permissions and ownership of the missing contents inside  the `/etc/puppetlabs/code/environments/` file after they have been restored back.
```
